tests/acceptance/features/steps/keystone_token.py

```diff
@@ -100,7 +117,51 @@ def step_request_trial_users_list(context):
     :param context: Context of the acceptance test execution.
     :return: Nothing.
     """
-    context.expiredusers.get_list_trial_users()
+    context.expiredusers.get_trial_users()
+
+
+@when(u'I request a list of community users from the Keystone')
+@given(u'a list of community users from the Keystone')
+def step_request_community_users_list(context):
+    """
+    Request the list of trial users from the Keystone.
+    :param context: Context of the acceptance test execution.
+    :return: Nothing.
+    """
+    users = context.expiredusers.get_community_users()
+    print('\n      Number of community users found: {}\n\n'.format(len(users)))
```

Comment:
they are acceptance tests which show the information in the screen. The print was alredy there